### DISCUSSION TITLE
trans/integrating-dapr-with-cilium/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#354] [Announcements]
# trans/integrating-dapr-with-cilium/

本文探讨了将 Dapr 与 Cilium 集成的无 Sidecar 服务网格方法，强调了其高性能和安全性。

https://jimmysong.io/trans/integrating-dapr-with-cilium/

**Comment by rootsongjc** [comment]
Dapr 提供了共享的部署模型，可以将 Dapr 运行时部署为 DaemonSet 或 Deployment，解耦了运行时与应用的生命周期，有点像 Istio 的 Ambient mode，但是它有很多构建块，上手不容易。
